proposals/date_shift.json

```diff
@@ -0,0 +1,136 @@
+{
+    "id": "date_shift",
+    "summary": "Calculates and manipulates dates and times",
```

Comment:
This summary is bit over-promising I think. With "calculation" I would associate converting a date from a UNIX timestamp, and with "manipulate" I associate setting specific date items (e.g. "set day to 01 and time to 00:00:00").

proposal:

> Shift date/time a given amount

Reply:
My intention with the summaries is usually to explain the process id better in very short. 
That we are shifting dates here should be clear from the process id, but for users that don't get it, I'd like to give a different wording because otherwise it's just a longer version of the process id. So if we could find a version that doesn't use shift, I'd be more happy. :-)

Reply:
Vielleicht so?
```suggestion
    "summary": "Manipulates dates and times by addition or subtraction",
```